Parsers/ASimWebSession/Parsers/vimWebSessionBarracudaCEF.yaml

```diff
@@ -0,0 +1,288 @@
+Parser:
+  Title: Web Session ASIM filtering parser for Barracuda CEF
+  Version: '0.2.1'
+  LastUpdated: May 21 2024
+Product:
+  Name: Barracuda WAF
+Normalization:
+  Schema: WebSession
+  Version: '0.2.6'
+References:
+- Title: ASIM Web Session Schema
+  Link: https://aka.ms/ASimWebSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Logs type Description in Barracuda CEF
+  Link: https://campus.barracuda.com/product/webapplicationfirewall/doc/78810008/types-of-logs/
+Description: |
+  This ASIM parser supports filtering and normalizing Barracuda WAF logs ingested in 'CommonSecurityLog' table to the ASIM Web Session normalized schema.
+ParserName: vimWebSessionBarracudaCEF
+EquivalentBuiltInParser: _Im_WebSession_BarracudaCEF
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: srcipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: ipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: url_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: httpuseragent_has_any
+    Type: dynamic 
+    Default: dynamic([])
+  - Name: eventresultdetails_in
+    Type: dynamic 
+    Default: dynamic([])
+  - Name: eventresult
+    Type: string
+    Default: '*'
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let barracudaSchema = datatable(
```

Comment:
we don't need this schema for cef table parser

Reply:
Done

Reply:
@Alekhya0824  Please apply the same change to ASim file as well.
ASimWebSessionBarracudaWAF